CHANGELOG.md

```diff
@@ -2,6 +2,31 @@
 
 ## Unreleased
 
+### Fixes
```

Comment:
we have 2 `### Fixes` sections now, I guess we should merge them for this version